modules/nw-metallb-operator-custom-resources.adoc

```diff
@@ -20,13 +20,13 @@ An `IPAddressPool` requires a name.
 The documentation uses names like `doc-example`, `doc-example-reserved`, and `doc-example-ipv6`.
 An `IPAddressPool` assigns IP addresses from the pool.
```

Comment:
The controller assigns IP addresses from 'IPAddressPool' pool of addresses.

Reply:
Updated, thanks!